py/selenium/webdriver/remote/webdriver.py

```diff
@@ -163,10 +163,10 @@ class WebDriver(BaseWebDriver):
 
     def __init__(
         self,
-        command_executor="http://127.0.0.1:4444",
-        keep_alive=True,
-        file_detector=None,
-        options: Union[BaseOptions, List[BaseOptions]] = None,
+        command_executor: Union[str, RemoteConnection] = "http://127.0.0.1:4444",
+        keep_alive: bool = True,
+        file_detector: Optional[FileDetector] = None,
+        options: Optional[Union[BaseOptions, List[BaseOptions]]] = None,
```

Comment:
Adding `Optional` here is valid because the default is `None`, so this says that `None` can be given.

However:

* `None` is not mentioned in the `Args` section below : "instance of a driver options.Options class".
* Giving `None` will lead to an `AttributeError` because he code below will try `options.to_capabilities()`.

My suggestion is to merge this, and then consider how to remove the default, in a separate PR.

Reply:
👍